VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts

```diff
@@ -314,15 +320,15 @@ export abstract class WaitingRoomBaseComponent {
     }
 
     handleCallSetup(callSetup: CallSetup) {
-        this.logger.debug('[WR] - Conference has setup');
+        this.logger.debug(`${this.loggerPrefix} Conference has setup`);
```

Comment:
could you add the conference and participant as the third param like the other logs do please

Reply:
added conference id and participant id